src/config/config.go

```diff
@@ -23,22 +23,28 @@ import (
 	"io/ioutil"
 
 	"github.com/coreos/ignition/third_party/github.com/camlistore/camlistore/pkg/errorutil"
+	"github.com/coreos/ignition/third_party/github.com/coreos/go-semver/semver"
+)
+
+var (
+	MaxVersion = semver.Version{
+		Major:      2,
+		Minor:      0,
+		PreRelease: "dev",
```

Comment:
When does this get changed, to what, and by which mechanism?

Will we manually switch this to something else when the config is stable?